app/entry.server.tsx

```diff
@@ -148,26 +143,6 @@ function handleBrowserRequest(
   });
 }
 
-/**
- * Flushes pending Sentry events with a 2 second timeout and in a way that cannot create unhandled promise rejections.
- */
-async function flushSafelyWithTimeout(): Promise<void> {
-  try {
-    await Sentry.flush(2000);
-  } catch (error) {
-    console.error('Error while flushing events to Sentry', {
-      cause: error,
-    });
-  }
-}
-
-export const handleError: HandleErrorFunction = async (error, { request }) => {
-  // React Router may abort some interrupted requests, don't log those
-  if (!request.signal.aborted) {
-    Sentry.captureException(error);
-
-    console.error(error);
-
-    vercelWaitUntil(flushSafelyWithTimeout());
-  }
-};
+export const handleError = Sentry.createSentryHandleError({
```

Comment:
this is the function that they added to fix a bug I reported. that's why we don't need to implement it ourselves anymore and we can remove `@sentry/core` dep which was needed for `vercelWaitUntil`